tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedInputStreamImpl.java

```diff
@@ -116,7 +116,7 @@ public int read(byte[] b, int off, int len) throws IOException {
 
     @Override
     public void seek(long position) {
-        log.debug("Seeking to {} on {}/{}, current position {}", position, bucket, key, cursor);
+        log.info("Seeking to {} on {}/{}, current position {} (bufStart:{}, bufEnd:{})", position, bucket, key, cursor, bufferOffsetStart, bufferOffsetEnd);
```

Comment:
Can this be annoying?

Reply:
ah yes, should be debug

Reply:
@ivankelly can you change it to debug please?

Reply:
@ivankelly This is a very important change we are rooting for in 2.4.2. Merging this would be great.